app/controllers/katello/concerns/api/v2/repository_content_controller.rb

```diff
@@ -70,6 +70,8 @@ def compare
       end
 
       param :available_for, :string, :desc => N_("Return content that can be added to the specified object.  The values 'content_view_version' and 'content_view_filter are supported.")
+      param :show_all_for, :bool,
```

Comment:
Are show_all_for and available_for mutually exclusive? Would be helpful to raise an error if so